src/PooltogetherGovernor.sol

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+import {
+  Governor, GovernorCountingFractional
+} from "flexible-voting/src/GovernorCountingFractional.sol";
+import {
+  ERC20VotesComp,
+  GovernorVotesComp
+} from "@openzeppelin/contracts/governance/extensions/GovernorVotesComp.sol";
+import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
+import {
+  GovernorTimelockCompound,
+  ICompoundTimelock
+} from "@openzeppelin/contracts/governance/extensions/GovernorTimelockCompound.sol";
+import {GovernorSettings} from "@openzeppelin/contracts/governance/extensions/GovernorSettings.sol";
+
+/// @notice The upgraded Pooltogether Governor: Bravo compatible and built with OpenZeppelin.
+contract PooltogetherGovernor is
+  GovernorCountingFractional,
+  GovernorVotesComp,
+  GovernorTimelockCompound,
+  GovernorSettings
+{
+  /// @notice The address of the POOL token on Ethereum mainnet from which this Governor derives
+  /// delegated voting weight.
+  ERC20VotesComp private constant POOL_TOKEN =
+    ERC20VotesComp(0x0cEC1A9154Ff802e7934Fc916Ed7Ca50bDE6844e);
+
+  /// @notice The address of the existing Pooltogether DAO Timelock on Ethereum mainnet through
+  /// which
+  /// this Governor executes transactions.
+  ICompoundTimelock private constant TIMELOCK =
+    ICompoundTimelock(payable(0x42cd8312D2BCe04277dD5161832460e95b24262E));
+
+  /// @notice Human readable name of this Governor.
+  string private constant GOVERNOR_NAME = "Pooltogether Governor Bravo";
+
+  /// @notice The number of POOL (in "wei") that must participate in a vote for it to meet quorum
+  /// threshold.
+  ///
+  /// TODO: placeholder
+  uint256 private constant QUORUM = 2_500_000e18; // 2,500,000 POOL
+
+  /// @param _initialVotingDelay The deployment value for the voting delay this Governor will
+  /// enforce.
+  /// @param _initialVotingPeriod The deployment value for the voting period this Governor will
+  /// enforce.
+  /// @param _initialProposalThreshold The deployment value for the number of POOL required to
+  /// submit
+  /// a proposal this Governor will enforce.
+  constructor(
+    uint256 _initialVotingDelay,
+    uint256 _initialVotingPeriod,
+    uint256 _initialProposalThreshold
+  )
+    GovernorVotesComp(POOL_TOKEN)
+    GovernorSettings(_initialVotingDelay, _initialVotingPeriod, _initialProposalThreshold)
+    GovernorTimelockCompound(TIMELOCK)
+    Governor(GOVERNOR_NAME)
+  {}
+
+  /// @dev We override this function to resolve ambiguity between inherited contracts.
+  function supportsInterface(bytes4 interfaceId)
+    public
+    view
+    virtual
+    override(Governor, GovernorTimelockCompound)
+    returns (bool)
+  {
+    return GovernorTimelockCompound.supportsInterface(interfaceId);
+  }
+
+  /// @dev We override this function to resolve ambiguity between inherited contracts.
+  function castVoteWithReasonAndParamsBySig(
```

Comment:
This was added on top of the Gitcoin upgrade

Reply:
Hmm interesting, not totally clear to me why that is, but we can investigate.

Reply:
The error without this indicated it was implemented by both the `Governor` and `GovernorCountingFractional` contacts.

Reply:
I see why we're getting this now and it's not a problem 👍